.vscode/launch.json

```diff
@@ -0,0 +1,18 @@
+{
+  // Use IntelliSense to learn about possible attributes.
+  // Hover to view descriptions of existing attributes.
+  // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
```

Comment:
I understand that these comment lines are things vscode generates for you, but are you mad if I weill remove them since they are not valid in real JSON? :)

Reply:
removed